### PR TITLE
[8.0][FIX] stock_split_picking: do not write date_done when splitting closes

### PR DESCRIPTION
**Pull request by oihane**
issue #375

Proposed change to odoo / ocb
https://github.com/OCA/OCB/pull/661
https://github.com/odoo/odoo/pull/20131

### CODE REVIEW

File: stock_split_picking/model/stock.py [hunk 1]
```diff
@@ -53,6 +53,12 @@ def split_process(self):
             'context': ctx,
         }
 
+    @api.multi
+    def write(self, values):
+        if self.env.context.get('do_only_split') and 'date_done' in values:
+            del values['date_done']
```
Comment (rousseldenis) [inline stock_split_picking/model/stock.py:59]: @oihane Please add test

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

File: stock_split_picking/model/stock.py (hunk 1 above)
Comment (oihane) [reply stock_split_picking/model/stock.py:59]: @rousseldenis test added

**Review by rousseldenis — APPROVED** [review]
Code review.
@oihane Is this ok ?

**Comment by oihane** [comment]
:+1: perfect

**Comment by rousseldenis** [comment]
As very old, but tests are ok, merging this

**Pull request merged**